frame/support/src/dispatch.rs

```diff
@@ -1108,6 +1112,7 @@ macro_rules! decl_module {
 		{
 			fn on_finalize($param: $param_ty) {
 				$crate::sp_tracing::enter_span!("on_finalize");
+				$crate::wasm_tracing_span!(module_path!(), "on_finalize_wasm");
```

Comment:
Why is the name different and why can we not use the same `enter_span` on native and on wasm?

Reply:
yes, makes sense to match it, will change